src/main/webapp/app/lecture/lecture-unit/lecture-unit-management/video-unit-form/video-unit-form.component.html

```diff
@@ -50,58 +50,64 @@
                         formControlName="competencyLinks"
                     />
                 </div>
-                <div class="container" style="padding: 0">
-                    <div class="row">
-                        <div class="col-4">
-                            <label class="required" for="source">{{ 'artemisApp.videoUnit.createVideoUnit.source' | artemisTranslate }}*</label>
-                        </div>
-                        <div class="col-1"></div>
-                        <div class="col-7">
-                            <label for="urlHelper" jhiTranslate="artemisApp.videoUnit.createVideoUnit.urlVideoHelper"></label>
-                        </div>
-                    </div>
-                    <div class="row">
-                        <div class="col-4">
-                            <input
-                                type="text"
-                                class="form-control"
-                                id="source"
-                                formControlName="source"
-                                [placeholder]="'artemisApp.videoUnit.createVideoUnit.sourcePlaceHolder' | artemisTranslate"
-                            />
-                        </div>
-                        <div class="col-1 mx-auto text-center align-self-center">
-                            <button id="transformButton" type="button" class="btn btn-sm btn-primary" [disabled]="!isTransformable" (click)="setEmbeddedVideoUrl($event)">
-                                <fa-icon [icon]="faArrowLeft" />
-                            </button>
+                <div class="row" style="padding: 0">
```

Comment:
```suggestion
                <div class="row p-0">
```
please avoid css in html